plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/tasks/PostgreDatabaseBackupHandler.java

```diff
@@ -138,7 +138,9 @@ public void fillProcessParameters(
             log.debug("Can't find specific schemas/tables for the backup");
             return;
         }
-        if (!CommonUtils.isEmpty(arg.getTables())) {
+        if (settings.isCompleteBackup()) {
+            // dont add any restrictions
+        } else if (!CommonUtils.isEmpty(arg.getTables())) {
```

Comment:
Most IDEs consider such a statement as empty. Can you please put a `return` inside similar to the check above?